pallets/axelar-router/src/tests.rs

```diff
@@ -0,0 +1,8 @@
+use crate::mock::*;
+
+#[test]
+fn configure_router() {
+	new_test_ext().execute_with(|| {
+		//TODO
```

Comment:
Are you OK with adding unit/integration tests after we eventually merge this? Would love to have these changes in the gateway.

Reply:
Will try to have it for tomorrow @cdamian 🤞🏻